### DISCUSSION TITLE
connection failure [Apr 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1819] [Discord Threads]
> 🙋‍♂️ **Priyanka**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### make sure your db is up and running  
#  
> 👨‍🚀 **wingkwong**  
#### if u r using docker, the host would be host.docker.internal  
#  
> 🙋‍♂️ **Priyanka**  
#### my db is running but still facing the issue  
#  
> 🙋‍♂️ **Harbard**  
#### Is it running on the same computer ? Are you using docker ?  
#  
> 🙋‍♂️ **Priyanka**  
#### it is running on the same computer  
#  
> 🙋‍♂️ **Harbard**  
#### How did you install/launch your MySQL db and nocoDB ?  
#  
> 🙋‍♂️ **Harbard**  
#### What do you mean with "on the web" ?  
#  
> 🙋‍♂️ **Priyanka**  
#### I haven't  installed the nocodb. I'm using it on web  
#  
> 🙋‍♂️ **Priyanka**  
#### I don't know how to install it. Can you please help me  
#  
> 🙋‍♂️ **Priyanka**  
#### And I installed MySQL db from their website  
#  
> 👨‍🚀 **wingkwong**  
#### are u using localhost here?  
![image0](https://cdn.discordapp.com/attachments/963402632519647253/963698896679350272/unknown.png)  
#  
> 🙋‍♂️ **Priyanka**  
#### yes  
#  
> 👨‍🚀 **wingkwong**  
#### can u try "host.docker.internal" instead  
#  
> 🙋‍♂️ **Priyanka**  
#### did it.. but still the same error  
![image0](https://cdn.discordapp.com/attachments/963402632519647253/963700251351449630/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### can we connect via google meet?  
#  
> 🙋‍♂️ **Priyanka**  
#### sure  
#  
> 👨‍🚀 **wingkwong**  
#### good. lemme setup  
#  
> 👨‍🚀 **wingkwong**  
#### DMed u the link  
#  
> 🙋‍♂️ **Priyanka**  
#### ok thanks  
#  
> 👨‍🚀 **wingkwong**  
#### UPD: preliminary findings - cannot connect with root ac even locally. users will investigate the reasons.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963402632519647253)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by surendra2022** [comment]
Changing from `localhost` to `host.docker.internal` solved my issue

**Reply by KneePham** [reply]
works thanks
